frontend/src/localization/de.json

```diff
@@ -385,8 +385,8 @@
     "closeAll": "Alle Konzepte schließen"
   },
   "help": {
-    "tabConceptTrees": "Auswahl der Konzepte aus den Konzepten zur Verwendung in Editor oder Formular-Editor.",
-    "tabPreviousQueries": "Auswahl der getätigten Anfragen zur Verwendung in Editor, Zeit-Editor oder Formular-Editor.",
+    "tabConceptTrees": "Auswahl der Konzepte zur Verwendung im Editor oder Formular-Editor.",
+    "tabPreviousQueries": "Auswahl der getätigten Anfragen zur Verwendung im Editor, Zeit-Editor oder Formular-Editor.",
```

Comment:
Ob hier `in` oder `im` richtig ist kann ich nicht sagen. Es scheint beides zu gehen nur was damit ausgedrückt wird ist leicht unterschiedlich